Makefile

```
@@ -391,68 +394,63 @@ run-tests-cluster-vst-1.1-ssl:
 	@echo "Cluster server, Velocystream 1.1, SSL, with authentication"
 	@${MAKE} TEST_MODE="cluster" TEST_AUTH="rootpw" TEST_SSL="auto" TEST_CONNECTION="vst" TEST_CVERSION="1.1" __run_tests
 
+
+COMMON_DOCKER_CMD_PARAMS = \
+	--name=$(TESTCONTAINER) \
+	$(TEST_NET) \
+	-e TEST_ENDPOINTS=$(TEST_ENDPOINTS) \
+	-e TEST_NOT_WAIT_UNTIL_READY=$(TEST_NOT_WAIT_UNTIL_READY) \
+	-e TEST_AUTHENTICATION=$(TEST_AUTHENTICATION) \
+	-e TEST_JWTSECRET=$(TEST_JWTSECRET) \
+	-e TEST_MODE=$(TEST_MODE) \
+	-e TEST_BACKUP_REMOTE_REPO=$(TEST_BACKUP_REMOTE_REPO) \
+	-e TEST_BACKUP_REMOTE_CONFIG='$(TEST_BACKUP_REMOTE_CONFIG)' \
+	-e TEST_DEBUG='$(TEST_DEBUG)' \
+	-e TEST_ENABLE_SHUTDOWN=$(TEST_ENABLE_SHUTDOWN) \
+	-e ENABLE_DATABASE_EXTRA_FEATURES=$(ENABLE_DATABASE_EXTRA_FEATURES) \
+	-e GODEBUG=tls13=1 \
+	-e CGO_ENABLED=$(CGO_ENABLED)
+
+
 # Internal test tasks
 __run_tests: __test_debug__ __test_prepare __test_go_test __test_cleanup
 
+
+ALL_DOCKER_CMD_PARAMS=\
```

```suggestion
ALL_DOCKER_CMD_PARAMS=\
```

DOCKER_V1_CMD_PARAMS=\